pywbem/_nocasedict.py

```diff
@@ -368,19 +368,28 @@ def __repr__(self):
 
     def update(self, *args, **kwargs):
         """
-        Update the dictionary from sequences of key/value pairs provided in any
-        positional arguments, and from key/value pairs provided in any keyword
-        arguments. The key/value pairs are not copied.
+        Update the dictionary from key/value pairs. If an item for a key
+        exists in the dictionary, its value is updated. If an item for a key
+        does not exist, it is added to the dictionary at the end.
+        The provided keys and values are not copied.
```

Comment:
This sentence takes some thinking.  We mean that the values from the dictionary are used without copying them.

Reply:
DONE.
Changed the sentence to: "The provided keys and values are stored in the dictionary without being copied."